deployment/helm/values.yaml

```diff
@@ -84,6 +86,75 @@ agent:
   allowDebugEcho: false
   # linuxOnly dictates whether the Akri Agent will only run on a linux node
   linuxOnly: true
+  # registrationPortName is the name of the of the port the registration service is on
+  registrationPortName: registration
+  registrationPort: 10000
+
+customDiscovery:
+  # enabled defines whether to load a customDiscovery configuration
+  enabled: false
+  # name is the Kubernetes resource name that will be created for this
+  # customDiscovery configuration
+  name: akri-custom-discovery
+  # protocolName is the name of the protocol the Discovery Handler registers under
+  protocolName: 
+  # properties is a map of properties that will be passed to any instances
+  # created as a result of applying this customDiscovery configuration
+  properties:
+  # capacity is the capacity for any instances created as a result of
+  # applying this customDiscovery configuration
+  capacity: 1
+  # discoveryDetails is the map of discovery details that is 
+  # passed to the custom Discovery Handler
+  discoveryDetails: {}
+  # shared defines whether instances created as a result of
+  # applying this customDiscovery configuration as shared
+  shared: true
+  brokerPod:
+    # container used by customDiscovery
+    image:
```

Comment:
should we have `tag` here too?

Reply:
should this be:

```
image:
  repository:
  tag:
  pullPolicy:
```

Reply:
since it is someones custom image, i assumed breaking it up would just be more for them to configure when using the chart. From my understanding, we broke it up for our Akri images so we could configure the tag with the appropriate version.

Reply:
i think consistency is helpful.  our helm charts specify containers one way everywhere else ... it seems like it should do the same here.

Reply:
For brokerPods, while we have the `tag:` value in some of the protocol's templates (udev and debug echo dont have tag values), none of them use the tag value in the template and accordingly none of the documentation specifies setting it. Like you're saying, some pattern would be great. It seems that either (1) specify tags for Akri built images or (2) always specify tags or (3) only specify tags for akri core components (Controller, Agent, Webhook, and DHs??). Making this consistent may be best suited for another pr since if we do 1 or 2, docs and e2e tests will need to change

Reply:
i think we should have the tag everywhere.  it is fine to have defaults (as we do) that default to the helm chart version so that it isn't required most of the time.

Reply:
fixed in 4b4b441fc9215